index.test.js

```diff
@@ -216,6 +216,20 @@ describe('validate-commit-msg.js', function() {
       expect(errors).to.deep.equal(['INVALID COMMIT MSG: subject does not match subject pattern!']);
       expect(logs).to.deep.equal([msg]);
     });
+
+    it('should lowercase type when autoFix is true and make it valid', function() {
+      m.config.autoFix = true;
```

Comment:
Like I said, I would definitely recommend Jest over AVA these days. See [this](https://medium.com/@kentcdodds/migrating-to-jest-881f75366e7e) for more details.